File: PV239/seminar5/src/main/java/cz/pv239/seminar5/fragment/MainFragment.java
package cz.pv239.seminar5.fragment;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.core.app.Fragment;
import butterknife.ButterKnife;
import butterknife.Unbinder;
import cz.pv239.seminar5.R;

public class MainFragment
        extends Fragment {

    private static final String TAG = MainFragment.class.getSimpleName();

    private Unbinder mUnbinder;

    public static MainFragment newInstance() {
        return new MainFragment();
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.fragment_main, container, false);
        mUnbinder = ButterKnife.bind(this, view);

        return view;
    }

    @Override
    public void onDestroyView() {
        super.onDestroyView();
        mUnbinder.unbind();
    }
}

File: PV239/seminar5/src/main/java/cz/pv239/seminar5/fragment/MainFragment.kt
package cz.pv239.seminar5.fragment

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import cz.pv239.seminar5.R

class MainFragment : Fragment() {

    companion object {

        fun newInstance(): MainFragment {
            return MainFragment()
        }
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_main, container, false)
    }
}
